### PR TITLE
Update _bursaries-and-scholarships.md

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card
https://trello.com/c/7xcL3WkG/742-move-bursaries-scholarships-content-further-up-the-page-tabulate
### Context
The Bursaries & Scholarships information on the FUNDING page is popular but is currently not as prominent as it could be. Needs to be pushed higher up the page. 

Also candidates often use both to compare and contrast. We should tabulate the information so candidates can better see them side by side. 

### Changes proposed in this pull request
Moved tabulated bursaries and scholarships info on to funding page. Updated CTAs. Removed some duplicate content.

### CODE REVIEW

File: content/guidance/financial-support-for-teacher-training/_bursaries-cta.html.erb [hunk 1]
```diff
@@ -1 +1,3 @@
-<%= render CallsToAction::ChatOnlineComponent.new %>
+<%= render CallsToAction::ChatOnlineComponent.new(
+    text: "If you're unsure about whether you could be entitled to a bursary or scholarship, you can chat to us.",
```
Comment (peteryates) [inline content/guidance/financial-support-for-teacher-training/_bursaries-cta.html.erb:2]: Passing custom text into the `ChatOnlineComponent` isn't currently supported. I will create a PR to support this shortly.

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/components/calls_to_action/chat_online_component.html.erb

File: content/funding-your-training/_bursaries-and-scholarships-cta.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <%= render CallsToAction::SimpleComponent.new(
   icon: "icon-arrow",
-  text: "Find out how much is awarded for training to teach certain subjects through bursaries and scholarships.",
+  text: "Find out more information about bursaries and scholarships.",
```
Comment (peteryates) [inline content/funding-your-training/_bursaries-and-scholarships-cta.html.erb]: Do you 'find out information' or 'find information'?

Comment (MylesJarvis) [reply content/funding-your-training/_bursaries-and-scholarships-cta.html.erb]: Trying to get across that it's more than we've shown in this table... 

Find further information?

**Review by peteryates — APPROVED** [review]
🚢

**Pull request merged**